### PR TITLE
qa- classes removed from scss, they are selectors for e2e tests

### PR DESCRIPTION
**Pull request by aditudorache**

### CODE REVIEW

File: src/app/components/DataSelection/DataSelection.jsx [hunk 1]
```diff
@@ -50,7 +50,7 @@ const DataSelection = ({
 
   return (
     <div className="c-data-selection c-dashboard__content">
-      <div className="qa-data-selection-content">
+      <div className="c-data-selection-content qa-data-selection-content">
```
Comment (royderks) [inline src/app/components/DataSelection/DataSelection.jsx]: The removed `qa-data-selection-content` is still here? Replace with `data-test`?

File: src/app/components/DataSelection/__snapshots__/DataSelection.test.jsx.snap [hunk 2]
```diff
@@ -5,7 +5,7 @@ exports[`DataSelection should render LoadingIndicator 1`] = `
   className="c-data-selection c-dashboard__content"
 >
   <div
-    className="qa-data-selection-content"
+    className="c-data-selection-content qa-data-selection-content"
```
Comment (royderks) [inline src/app/components/DataSelection/__snapshots__/DataSelection.test.jsx.snap]: Same as previous

File: src/app/components/DataSelection/DataSelection.jsx (hunk 1 above)
Comment (aditudorache) [reply src/app/components/DataSelection/DataSelection.jsx]: out of scope

File: src/app/components/DataSelection/__snapshots__/DataSelection.test.jsx.snap (hunk 2 above)
Comment (aditudorache) [reply src/app/components/DataSelection/__snapshots__/DataSelection.test.jsx.snap]: out of scope

**Pull request merged**